contracts/src/token/erc721/extensions/metadata.rs

```diff
@@ -21,6 +21,9 @@ sol_storage! {
     }
 }
 
+// TODO: IErc721Metadata should be refactored to have same api as solidity
```

Comment:
Remove this comment. We were facing this issue with `v0.1` and decided internally that this is the best way for implementing this extension.